docs/content/dagster-plus/managing-deployments/branch-deployments/using-branch-deployments-with-github.mdx

```diff
@@ -30,87 +33,108 @@ To complete the steps in this guide, you'll need:
 
-<Note>
-  If you're using Serverless deployment, this step isn't required.{" "}
-  <a href="#step-42-configure-github-action-secrets">Skip to the next step</a>.
-</Note>
+   For example, for an Amazon ECR registry, you'd uncomment the following portion of the file:
+
+   ```yaml
+   jobs:
+     dagster-cloud-deploy:
+       steps:
+         - name: Configure AWS credentials
+           uses: aws-actions/configure-aws-credentials@v1
+           with:
+             aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+             aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+             aws-region: ${{ secrets.AWS_REGION }}
+   ```
+
+4. Save and commit the files to the repository.
+
+</TabItem>
+<TabItem name="Serverless">
+
+### Serverless
 
-In the [`dagster_cloud.yaml` file](/dagster-plus/managing-deployments/dagster-cloud-yaml), replace `build.registry` with the registry used by the [agent you created in Step 2](#step-2-create-and-configure-an-agent).
+Add the following files to your project in a `.github/workflows` directory:
 
-For example:
+- [`branch_deployments.yml`](https://github.com/dagster-io/dagster-cloud-serverless-quickstart/blob/main/.github/workflows/branch_deployments.yml)
+- [`deploy.yml`](https://github.com/dagster-io/dagster-cloud-serverless-quickstart/blob/main/.github/workflows/deploy.yml)
 
-```yaml
-# dagster_cloud.yaml
+</TabItem>
+</TabGroup>
 
-locations:
-  - location_name: example_location
-    code_source:
-      python_file: repo.py
-    build:
-      directory: ./example_location
-      registry: 764506304434.dkr.ecr.us-east-1.amazonaws.com/branch-deployment-agent
-```
+---
 
-### Step 4.2: Configure GitHub Action secrets
+## Step 4: Configure the GitHub repository
+
+In this section, you'll:
+
+1. [Configure GitHub Action secrets](#step-41-configure-github-action-secrets)
+2. [Verify Action runs](#step-42-verify-action-runs)
+
+### Step 4.1: Configure GitHub Action secrets
 
 <Note>
-  If you used the GitHub app to add the repository as a code location, this step{" "}
-  isn't required.
-  <br></br>
-  <br></br>
-  <strong>Want to use secrets in your Dagster code?</strong> Check out the <a href="/dagster-plus/managing-deployments/environment-variables-and-secrets">
-    Dagster+ environment variables and secrets guide
-  </a> for more info.
+  <strong>If you used the Dagster+ GitHub app to import your project</strong>,
+  this step was already done for you.{" "}
+  <a href="#step-5-access-the-branch-deployment">Skip to the next step</a>.
 </Note>
 
+In this step, you'll add the required secrets to the GitHub repository. **Note**: This isn't how environment variables and secrets can be used in Dagster code. Refer to the [Environment variables and secrets guide](/dagster-plus/managing-deployments/environment-variables-and-secrets) for more information.
+
 <AddGitubRepositorySecret />
 
 **For Hybrid deployments,** repeat steps 3-6 for each of the secrets required for your registry type:
```

Comment:
For Serverless, users should only need to set `DAGSTER_CLOUD_API_TOKEN` AND `ORGANIZATION_ID`, correct?